loopchain/blockchain/votes/v1_0/vote_factory.py

```diff
@@ -0,0 +1,128 @@
+from lft.consensus.messages.data import DataPool
+from lft.consensus.messages.vote import Vote, VoteFactory, VoteVerifier
+
+import loopchain.utils as util
+from loopchain.blockchain.invoke_result import InvokeResultPool, InvokeResult
+from loopchain.blockchain.types import Hash32, ExternalAddress, Signature
+from loopchain.crypto.hashing import build_hash_generator
+from loopchain.crypto.signature import Signer
+from .vote import BlockVote
+
+hash_generator = build_hash_generator(1, "icx_vote")
+
+
+class BlockVoteFactory(VoteFactory):
+    def __init__(self, data_pool: DataPool, invoke_result_pool: InvokeResultPool, signer):
+        self._data_pool: DataPool = data_pool
```

Comment:
LFT does not use `blockHeight` which is intended to be derived from `datapool`.
So `BlockVoteFactory` has no reason to have `data_pool`.